src/common/blib_func.c

```diff
@@ -2781,6 +2781,51 @@ void cmd_genfunc(long funcCode, var_t *r) {
       mat_tov(r, m1, n, n, 1);
       free(m1);
     }
+  }
+    break;
+    //
+    // array <- TRANSPOSE(a)
+    //
+  case kwTRANSPOSE: {
+    int32_t rows, cols, pos1, pos2;
+    var_t *e;
+
+    v_init(r);
+    var_t *a = par_getvarray();  
+    IF_ERR_RETURN;    
+    
+    if (v_maxdim(a) > 2) {
+      err_matdim();
+      IF_ERR_RETURN;
+    }
+    
+    rows = ABS(v_lbound(a, 0) - v_ubound(a, 0)) + 1;
+
+    if (v_maxdim(a) == 2) {
+      cols = ABS(v_lbound(a, 1) - v_ubound(a, 1)) + 1;
+    } else {
+      cols = rows;
+      rows = 1;
+    }
+    
+    var_num_t *m = (var_num_t *)malloc(((rows) * (cols)) * sizeof(var_num_t));
+    
+    for(int32_t x = 0; x < cols; x++) {
```

Comment:
just  a super-nit pik. could u please put a space after `for` ie `for (`. same for the `if(` line below.

Reply:
No problem. Space added.